content/applications/finance/fiscal_localizations/hong_kong.rst

```diff
@@ -51,5 +61,389 @@ QR-code` option to *EMV Merchant-Presented QR-code*.
+
+.. image:: hong_kong/hk-individual-payslip.png
+   :alt: Hong Kong Individual Payslip
+
+.. note::
+   If the work entry for an employee was amended, click the :guilabel:` ⚙ (gear)` icon, then click 
+   :guilabel:`Recompute Whole Sheet` to refresh the payslip's **Worked Day & Inputs** section.
+
+The :guilabel:`Salary Computation` tab shows the detailed breakdown of the computation based on
+the salary rules configured for each structure type.
+
+.. image:: hong_kong/hk-salary-computation.png
+   :alt: Hong Kong Salary computation
+
+#. :guilabel:`Rent Allowance`: Amount derived from the employee's active rental record.
+#. :guilabel:`Basic Salary`: Amount of base salary provided (after rent allowance deduction)
+#. :guilabel:`713 Gross`: Net payable amount considering Commission, Internet Allowance, Reimbursements, 
+   Back-pay, Deduction, etc.
+#. :guilabel:`MPF Gross`: Net payable amount from 713 gross after consideration of additional allowances, 
+   deductions and end-of-year payment.
+#. :guilabel:`Employee Mandatory Contribution`: Employee MPF Contribution
+#. :guilabel:`Employer Mandatory Contribution`: Employer MPF Contribution
+#. :guilabel:`Gross`: Net payable amount from MPF gross after consideration of MPF deductions.
+#. :guilabel:`Net Salary`: Final payable amount to be paid to the employee.
+
+.. note::
+   There are no MPF contributions for the first month. Both **employee** and **employer**
+   contribution starts on second month and the **employer** contribution for the second month is
+   include in the first month as back-pay.
+
+Under the :guilabel:`Other Inputs` tab at the bottom of payslip, there are additional manual input
+types that are specific to HK Payroll:
+
+- :guilabel:`Back Pay`: Additional salary payout can be included under this category.
+- :guilabel:`Commission`: The commission earned during the period can be manually entered here.
+- :guilabel:`Global Deduction`: A lump-sum deduction from the entire payslip.
+- :guilabel:`Global Reimbursement`: A lump-sum reimbursement to the entire payslip.
+- :guilabel:`Referral Fee`: The additional bonus offered for any form of business-related referral.
+- :guilabel:`Moving Daily Wage`: To override the :abbr:`ADW (Average Daily Wage)` value used for leaves computation.
+- :guilabel:`Skip Rent Allowance`: If set, the rental allowance is excluded from the current payslip.
+- :guilabel:`Custom Average Monthly Salary`: To override the average monthly salary used for end-of-year payment.
+
+Once the payslips are ready, click :guilabel:`Create Draft entry` to generate a draft journal entry 
+found in the :guilabel:`Other Info` tab of the payslip.
+
+Paying employees
+----------------
+
+Once the draft journal entries have been posted, the company can now pay the employees.
+The user can choose between **two** different **payment methods**.
+
+- From the employee's payslip (:menuselection:`Payroll app --> Payslips`), once the payslip's journal
```

Comment:
to pay? all payslips? batches? The lat part of this direciton needs to be added.